cmake/Modules/Documentation.cmake

```diff
@@ -1,10 +1,9 @@
 ###############################################################################
 # Build documentation
 ###############################################################################
-option(BUILD_DOC "Build LAMMPS documentation" OFF)
+option(BUILD_DOC "Build LAMMPS HTML documentation" OFF)
 if(BUILD_DOC)
   include(ProcessorCount)
```

Comment:
If `NPROC` isn’t used, I think we can drop this too?

Reply:
yes